From 27b63a883efdb4d60666a8fa4edb8c82bc318f30 Mon Sep 17 00:00:00 2001
From: Giovanni Ricca <giovanniricca@proton.me>
Date: Tue, 9 Sep 2025 22:38:17 +0200
Subject: [PATCH 1/2] CameraUtil: Stub out `setScreenFps`

I CameraUtil: setScreenFps failed:java.lang.ClassNotFoundException: com.zte.performance.refreshrate.ScreenRefreshRateManager
D AndroidRuntime: Shutting down VM
E AndroidRuntime: FATAL EXCEPTION: main
E AndroidRuntime: Process: com.android.camera, PID: 10922
E AndroidRuntime: java.lang.RuntimeException: java.lang.ClassNotFoundException: com.zte.performance.refreshrate.ScreenRefreshRateManager
E AndroidRuntime:        at cn.nubia.camera.utils.CameraUtil.setScreenFps(CameraUtil.java:2038)
E AndroidRuntime:        at cn.nubia.camera.fragment.MoreSetting.exitMoreSettingsFragment(MoreSetting.java:890)
E AndroidRuntime:        at cn.nubia.camera.fragment.MoreSetting.pause(MoreSetting.java:129)
E AndroidRuntime:        at cn.nubia.camera.fragment.MemberFragment.pauseMoreSetting(MemberFragment.java:3039)
E AndroidRuntime:        at cn.nubia.camera.fragment.MemberFragment.onPause(MemberFragment.java:563)
E AndroidRuntime:        at cn.nubia.camera.normal.NormalMemberFragment.onPause(NormalMemberFragment.java:427)
---
 smali_classes2/cn/nubia/camera/utils/CameraUtil.smali | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/smali_classes2/cn/nubia/camera/utils/CameraUtil.smali b/smali_classes2/cn/nubia/camera/utils/CameraUtil.smali
index e170de81..f34a211e 100644
--- a/smali_classes2/cn/nubia/camera/utils/CameraUtil.smali
+++ b/smali_classes2/cn/nubia/camera/utils/CameraUtil.smali
@@ -8157,6 +8157,8 @@
 .method public static setScreenFps(Z)V
     .locals 7
 
+    return-void
+
     .line 2095
     invoke-static {}, Lcn/nubia/camera/nubiacameraconfig/Config;->getIntance()Lcn/nubia/camera/nubiacameraconfig/Config;
 
-- 
2.55.0

